pbench_logparse/README.md

```diff
@@ -0,0 +1,32 @@
+# pbench_logparse
+
+This golang tool was created to parse the output of time series data in CSV.
+
+```
+Usage of ./pbench_logparse:
+  -blkdev value
+        List of block devices
+  -i string
+        pbench run result directory to parse (default "/var/lib/pbench-results/test1/tools-default/")
```

Comment:
The idea is to provide an example directory, having it doesn't hurt anything but it exists for documentation.